### ISSUE TITLE
Color problem in plot_forest when comparing data

### ISSUE THREAD

**Issue by GWeindel** [arviz-devs/arviz#614]
**Describe the bug**
The shading used to group the parameters of compared models is confusing in forest_plot() 

**To Reproduce**
```
data = az.load_arviz_data('centered_eight')
data2 = az.load_arviz_data('centered_eight')

_ = az.plot_forest(data=[data, data2], model_names=['reduced', 'full'],  rope=(0,0), combined=True, figsize=(10, 6), credible_interval=0.95 )
```
![demo](https://user-images.githubusercontent.com/28750455/54210915-ae59e200-44e0-11e9-8613-99a142dc2ef6.png)

The expected behavior is one parameter (for both models) = one color

arviz 0.3.3

**Comment by aloctavodia** [comment]
Hi @GWeindel thanks for taking the time to report this issue. I am not able to reproduce this on master or with arviz 0.3.3.

**Comment by GWeindel** [comment]
Weird, I have it with the master or 0.3.3. As said in #613 there is some randomness in the generation of the shaded area, did you try to restart the console & run the code several times ?

**Comment by aloctavodia** [comment]
still no problem with the shade :-(

**Comment by canyon289** [comment]
I can try reproducing this error in ~13 hours for another check as well.

@GWeindel What version of the python and matplotlib are you using?

**Comment by GWeindel** [comment]
I looked it up and it is again python 3.5 specific. Should I fix it or isn't it worth it given the short life time left to python 3.5 ?

**Comment by ahartikainen** [comment]
Yes, we still fix Python 3.5 stuff.
